inductiva/resources/machines.py

```diff
@@ -42,6 +42,7 @@ def __init__(
                          data_disk_gb=data_disk_gb,
                          register=register)
         self.num_machines = num_machines
+        self._current_machines = 0
```

Comment:
maybe rename to `_active_machines` and add a comment about this being the number of active running machines
at the time of the request?

```suggestion
        self._active_machines = 0
```